### PR TITLE
Create new walkthrough tooltip component

### PR DESCRIPTION
**Pull request by sdzharkov**
This file modifies the tooltip component (copy-pasta) but multiple modifications. I prefer to have a separate file because it changes a lot of tooltip functionality.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -71,6 +73,7 @@ export {
   Overlay,
   ThemeProvider,
   ThemeConsumer,
+  WalkthroughTooltip,
```
Comment (zeulb) [inline src/index.js:76]: Good call making a new tooltip

File: src/tooltip/WalkthroughTooltip.js [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+
+import { ViewPropTypes, withTheme } from '../config';
+import { ScreenWidth, ScreenHeight, isIOS } from '../helpers';
+
+import Triangle from './Triangle';
+import getTooltipCoordinate, {
+  getElementVisibleWidth,
+} from './getTooltipCoordinate';
+
+class Tooltip extends React.PureComponent {
```
Comment (zeulb) [inline src/tooltip/WalkthroughTooltip.js]: nit: rename component name

**Review by zeulb — APPROVED** [review]

File: src/tooltip/WalkthroughTooltip.js (hunk 2 above)
Comment (sdzharkov) [reply src/tooltip/WalkthroughTooltip.js]: O man

**Review by jalhadi — APPROVED** [review]

File: src/tooltip/WalkthroughTooltip.js [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+            backgroundColor: highlightColor,
+            overflow: 'visible',
+            width: elementWidth,
+            height: elementHeight,
+          }}
+        >
+          <TouchableOpacity
+            onPress={onPress}
+            activeOpacity={1}
+            style={{
```
Comment (zeulb) [inline src/tooltip/WalkthroughTooltip.js:140]: `style={[{ flex: 1 }, renderedContainerStyles]}` ?

**Pull request merged**